### DISCUSSION TITLE
Docker image: Container stops immediately after startup

### DISCUSSION THREAD

**Discussion by ralphotowo** [shlinkio/discussions#68] [Help wanted] [answered]
#### How Shlink is set-up

* Shlink Version: stable (3.1.0)
* PHP Version: 8
* How do you serve Shlink: Docker image
* Database engine used: MySQL

#### Summary

I am attempting to test Shlink on my local machine using the following command:

`docker run --name shlink -p 8080:8080 -e DEFAULT_DOMAIN=localhost -e IS_HTTPS_ENABLED=true -e GEOLITE_LICENSE_KEY=kjh23ljkbndskj345 -e DB_DRIVER=mysql -e DB_HOST=35.XX.XX.XX -e DB_USER=root -e DB_PASSWORD=myDBpassw0rd shlinkio/shlink:stable`

The image correctly creates a new database "shlink" and successfully executes the migrations. After downloading the GeoLite2 db file, the container exits. My expectation is that the application should be accessible at the exposed port (localhost:8080). 

Did I miss something in the documentation? Anyone know why this is happening?

**Comment by acelaya** [comment, marked as answer]
What's the output of the image?

**Reply by ralphotowo** [reply]
@acelaya Thanks for your feedback. I tried this again to retrieve the output and it just worked. No idea what changed. I appreciate your help.
